to-do-app/src/components/Navbar.js

```diff
@@ -0,0 +1,14 @@
+import {Link} from 'react-router-dom'
+const Navbar=()=>{
+    return <div className="items nav">
```

Comment:
Should use <nav> element instead of div here.
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav